psutil/net.go

```diff
@@ -62,6 +62,17 @@ var netIOCounterLabels = map[string]label{
 	},
 }
 
+var netConntrackCounterLabels = map[string]label{
```

Comment:
what reason you didn't want to combine netConntrackCounterLables initialization with IO but rename the variable?